src/rotator_library/anthropic_compat/translator.py

```
@@ -9,10 +9,31 @@
-import json
 import uuid
 from typing import Any, Dict, List, Optional, Union
 
+# Try to import orjson for faster JSON handling (3-5x faster than stdlib json)
+try:
+    import orjson
+
+    def _json_dumps(obj) -> str:
+        return orjson.dumps(obj).decode('utf-8')
+
+    def _json_loads(s: str):
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `_json_loads`

(ANN202)